apache2/ag_mdb/ag_mdb.cpp

```diff
@@ -57,36 +56,91 @@ int Lock_create(struct agmdb_lock *new_lock, const char* lock_name, int lock_nam
         return AGMDB_ERROR_LOCK_WIN_NAME_INVALID_STRING;
     sprintf_s(read_lock_name, AGMDB_MAX_NAME_LEN, "DB_%s_LOCK_READ", lock_name);
     sprintf_s(write_lock_name, AGMDB_MAX_NAME_LEN, "DB_%s_LOCK_WRITE", lock_name);
-    new_mutex = CreateMutex(
```

Comment:
why do we change from createMutex, OpenMutex to just createMutex, try to get more context